sds_data_manager/lambda_code/SDSCode/dependency_config.py

```diff
@@ -23,6 +23,16 @@
         relationship="HARD",
         direction="DOWNSTREAM",
     ),
+    PreProcessingDependency(
+        primary_instrument="swapi",
+        primary_data_level="l0",
+        primary_descriptor="raw",
+        dependent_instrument="swapi",
+        dependent_data_level="l1",
+        dependent_descriptor="sci",
```

Comment:
Should this be `sci-1min`?

Reply:
Good catch. It should have been `all` instead.